modules/client-configuration/pages/package-management.adoc

```diff
@@ -47,3 +47,51 @@ When the profile has been created, you can use it to install more clients with t
 . In the {productname} {webui}, navigate to [guimenu]``Systems``, click the client to compare, and navigate to the menu:Systems[Software > Packages > Profiles] subtab.
     To compare with a stored profile, select the profile and click btn:[Compare].
 . To compare with another client, select the client name and click btn:[Compare] to see a list of differences between the two clients.
+
+
+
+== Orphaned Packages
+
+Orphaned packages do not have associated channels.
+Therefore, orphaned packages are usually not available to {productname} clients and cannot install such packages.
+
+Orphaned packages can exist as a result of one of the following events:
+
+
+* A repository removes a package.
+  By default, in {productname} 5.0, we also unlink such package from the channel, but do not remove it.
+  In {productname} 4.3, this was the strict mode behavior.
+* A repository supersedes a package with a newer version and thus removes the previous version.
+* The reposync process got interrupted (for example, because of an out of space exception) and thus downloaded packages were not associated with a channel.
+* Users manually uploaded packages, but did not associate them with any channel.
+
+Orphaned packages costs space in user environment and cannot easily
+distributed to clients because they are not associated with a channel.
```

Comment:
```suggestion
Orphaned packages cost space in user environment and cannot be easily distributed to clients because they are not associated with a channel.
```